### PR TITLE
(DOCSP-13767): Upgrade a Shared Cluster

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13767

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-13767-upgrading-shared-clusters/reference/upgrade-shared-cluster

### CODE REVIEW

**Comment by mongomoe** [comment]
To the reviewer: ignore all the test commits, it's going to be squashed and merged anyway. I just kept on getting build failures.

File: source/includes/steps-upgrade-shared-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+title: Shutdown All Running Emulators
+ref: add-the-sign-in-with-apple-capability
+content: |
+   Before upgrading your Cluster end any client application's Sync Connection.
```
Comment (cbush) [inline source/includes/steps-upgrade-shared-cluster.yaml:4]: Reading "end" here blended into the first clause ("upgrade your cluster end, ..."). Comma suggested (or use another word like "close").
```suggestion
   Before upgrading your cluster, end any client application's sync connection.
```

File: source/includes/steps-upgrade-shared-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   To do this shutdown all emulators, simulators, and terminals that are running
```
Comment (cbush) [inline source/includes/steps-upgrade-shared-cluster.yaml:5]:
```suggestion
   To do this, shut down all emulators, simulators, and terminals that are running
```

File: source/includes/steps-upgrade-shared-cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   your {+realm+} application. If you are running an iOS application, make sure
+   to `clean out your iOS simulator
+   <https://stackoverflow.com/questions/22075805/how-can-i-completely-clean-out-the-ios-simulator/22301054>`_
+   before shutting it down.
```
Comment (cbush) [inline source/includes/steps-upgrade-shared-cluster.yaml:9]: Why is this required and what if the app was already released so we can't know who has it running?

File: source/includes/steps-upgrade-shared-cluster.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+---
+title: Terminate Realm Sync
+ref: terminate-realm-sync
+content: |
+  Terminate Realm Sync in your {+realm+} Application to stop syncing data across
```
Comment (cbush) [inline source/includes/steps-upgrade-shared-cluster.yaml:14]:
```suggestion
  Terminate Realm Sync in your {+realm+} application to stop syncing data across
```

File: source/includes/steps-upgrade-shared-cluster.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
+title: Shutdown All Running Emulators
```
Comment (cbush) [inline source/includes/steps-upgrade-shared-cluster.yaml:1]: "Shutdown" is a noun. "Shut down" is the verb phrase.
```suggestion
title: Shut Down All Running Emulators
```

**Review by cbush — APPROVED** [review]
Looks good with a couple suggestions and a question. I would have Drew take a look at this as well.

File: source/includes/steps-upgrade-shared-cluster.yaml (hunk 3 above)
Comment (mongomoe) [reply source/includes/steps-upgrade-shared-cluster.yaml:9]: Just clarified with mike obrien and it's not needed, terminating/re-initializing sync will force the devices to also do a client resync (drop their view of the dataset and re-download everything). This is true for both prod and development, so we can just remove this line.
